doc/lua_api.txt

```diff
@@ -4347,6 +4348,8 @@ Utilities
           object_step_has_moveresult = true,
           -- Whether get_velocity() and add_velocity() can be used on players (5.4.0)
           direct_velocity_on_players = true,
+          -- New set_sky() syntax, including set_sun and set_moon functions
```

Comment:
since modders can test if set_sun is available this is not strictly needed, is it?

Reply:
Right. I added this one to see whether a feature or a run-time check is preferred. If there are no objections from modders I'll revert this one and point out that the presence of `set_sun` or `set_moon` should be checked instead.

Reply:
Yeah that'd be my preferred approach.